server/controllers/peopleController.ts

```diff
@@ -17,14 +19,19 @@ export default class PeopleController {
       if (crn) {
         try {
           const person = await this.personService.findByCrn(req.user.token, crn)
-          const application = await this.applicationService.createApplication(req.user.token, person.crn)
 
+          if (!isFullPerson(person)) {
+            throw new RestrictedPersonError(crn)
```

Comment:
so, in this case, do we always want to prevent the user from continuing? Or do we want to distinguish between a restricted person that the referrer has permission to refer, and a restricted person that the referred doesn't have permission to refer? I think this is cas1's approach: https://trello.com/c/IdrDyeuM/309-cannot-list-bookings-from-premises-with-lao-offenders-%F0%9F%90%9E

Reply:
good question - we'll need to raise it with HMPPS to confirm and come back to this - for now I'm assuming we don't let the application be created if it's an LAO

Reply:
slack thread here https://mojdt.slack.com/archives/C04T6SC31HA/p1692959904971339?thread_ts=1692959679.970929&cid=C04T6SC31HA

Reply:
ok thanks 👍